.github/workflows/build.yml

```diff
@@ -174,7 +175,7 @@ jobs:
     strategy:
       matrix:
         goos: [ darwin ]
-        goarch: [ "amd64" ]
+        goarch: [ "amd64", "arm64" ]
```

Comment:
darwin/arm64 was only introduced in 1.11.x

Reply:
Ah, ok. I was wondering why it was left out of these older release series. Is there a technical reason why it is 1.11.0 and beyond?

Reply:
Nothing technical; the decision was simply to support darwin/arm64 moving forwards, since it doesn't make sense to do a patch release introducing a new architecture as a backport

Reply:
100%!